src/skin/js/firstRun.js

```diff
@@ -0,0 +1,19 @@
+(function($) {
+
+$(window).on("load", function () {
+
+  function setSeenComic() {
+    var badger = chrome.extension.getBackgroundPage().badger;
+    var settings = badger.storage.getBadgerStorageObject("settings_map");
+    settings.setItem("seenComic", true);
```

Comment:
Do we want to do this with messaging? Otherwise I don't think it will work in private browsing (though I'm not sure whether it should).

Reply:
I don't think it matters in this scenario ... If it did matter, we should open a new issue since this would be an existing bug (shouldn't confuse this PR by tacking fixes onto it). But I think what happens is that the only way to open the walkthrough in Private Browsing is to via a Private Browsing window's popup, and that means you went through the reminder, which updates `seenComic` (or you didn't, which means `seenComic` has already been updated).

Reply:
oh yeah, didn't realize the ? button will update `seenComic` on its own. In that case, I don't think we need the change at all.